app/controllers/admin/teams_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Admin::TeamsController < ApplicationController
+  before_action :find_components, only: %i[show]
+  def index
+    @teams = Team.all
+  end
+
+  def new
+    @team = NewTeamEvent.new
+  end
+
+  def create
+    team_event = NewTeamEvent.create(team_params)
+    @team = team_event.team
+    if @team.valid? && team_event.valid?
+      redirect_to admin_teams_url
```

Comment:
Any reason we're using the absolute url instead of the relative path?
```suggestion
      redirect_to admin_teams_path
```